backend/.env.template

```
@@ -0,0 +1,9 @@
+DATABASE_URL=cockroachdb://your-username:your-password@your-host:26257/your-database?sslmode=require
+SECRET_KEY=your-secret-key-change-in-production
+
+# Cloudflare R2 Settings
+R2_ACCESS_KEY_ID=your-r2-access-key-id
+R2_SECRET_ACCESS_KEY=your-r2-secret-access-key
+R2_BUCKET_NAME=your-r2-bucket-name
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The R2_BUCKET_NAME key should go before the R2_SECRET_ACCESS_KEY key

(UnorderedKey)